src/main/java/cn/nukkit/block/BlockTallGrass.java

```diff
@@ -137,11 +139,15 @@ public boolean onActivate(@Nonnull Item item, Player player) {
                 };
             }
         }
-
-        if (dropSeeds) {
-            return new Item[]{
-                    new ItemSeedsWheat()
-            };
+        
+        Random random = new Random();
+        if (random.nextInt(7) == 0) {
```

Comment:
The base drop chance should be 87.5%, 1/7 gives 14.28%, it can be fixed changing the line to:
If using ThreadLocal:
```java
if (random.nextInt(8) <= 6) {
```

If using NukkitRandom:
```java
if (random.nextRange(0, 7) <= 6) {
```

7/8 = 87.5%

Reply:
Just updating that I was wrong, I miss interpreted this table, I thought the first row was one seed. But the 7/8 argument is still valid.
![image](https://user-images.githubusercontent.com/3679022/91649492-58b62400-ea4b-11ea-9356-37c2fc069c48.png)